text/3-entrees/currency-imbalances.md

```diff
@@ -0,0 +1,36 @@
+
+# Currency Imbalances
+
+`pallets/currency-imbalances`
+
+
+[`Imbalance`](https://substrate.dev/rustdocs/v2.0.0-rc4/frame_support/traits/trait.Imbalance.html) is used when tokens are burned or minted. In order to execute `imbalance` implement the [`OnUnbalanced`] (https://substrate.dev/rustdocs/v2.0.0-rc4/frame_support/traits/trait.OnUnbalanced.html)trait.
+In this pallet a specific amount of funds will be slashed from an account and 
+award a specific amount of funds to said specific account.
+
+Slash funds:
+pub fn slash_funds(origin, to_punish: T::AccountId, collateral: BalanceOf<T>) {
```

Comment:
When you paste a code block you need to mark it as a code block. It should look like this:

<pre>
```rust, ignore
fn example() {
  // your code here
  ()
}
```
</pre>

Reply:
Please check it because it is done.